ui/apps/notifications/src/components/pages/welcome-page.tsx

```diff
@@ -48,6 +64,17 @@ const WelcomePage: React.FC<RootComponentProps & IWelcomePageProps> = props => {
     });
   };
 
+  const connect = () => {
+    navigateTo?.({
+      appName: '@akashaorg/app-auth-ewa',
+      getNavigationUrl: (routes: Record<string, string>) => {
+        return `${routes.Connect}?${new URLSearchParams({
+          redirectTo: `${baseRouteName}`,
+        }).toString()}`;
+      },
+    });
+  };
```

Comment:
You can use the login modal. I see the bug with the modal is fixed on next.

Reply:
Hi Didd. In an earlier discussion, we talked about this and agreed that we would use a page with a connect button to direct the user to the connect page instead of using a modal. I implemented with a modal previously but changed to this :)

Reply:
It makes sense, I see you are connecting it with a button named Connect. Thanks for the clarity.